### ISSUE TITLE
Review purpose of getters

### ISSUE THREAD

**Issue by baileypierson** [NDBaxi/pwp-capstones#5]
https://github.com/NDBaxi/pwp-capstones/blob/a27bf19c6004d2c101160838943f2f423bfc87df/CapstoneProject/TomeRater.py#L60-L64

It is standard practice to have get functions return a parameter of an object, not a string

**Comment by NDBaxi** [comment]
Yup, I rechecked the instructions too.....they never asked me to return a string. Which means should it be simply as follows?

def get_title(self): 
   return  self.title
  
 def get_isbn(self): 
   return self.isbn
